### PR TITLE
temp: enable debug logging for the New Relic agent on stage

### PR DESCRIPTION
**Pull request by rgraber**
Following guidance at https://2u-internal.atlassian.net/wiki/spaces/SRE/pages/19254057/How+to+enable+New+Relic+Debug+and+Audit+logs. Necessary to debug some issues with the ticket mentioned in the comments.

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A SRE team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a SRE ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/spaces/SRE/pages/28967861/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/edxapp/templates/newrelic.ini.j2 [hunk 1]
```diff
@@ -27,3 +27,11 @@
 # `course_id`.
 #
 browser_monitoring.attributes.enabled=true
+
+# Temporarily turn on logging to debug https://github.com/openedx/event-bus-kafka/issues/45. If that ticket is closed,
+# the following three lines can be removed.
```
Comment (matthugs) [inline playbooks/roles/edxapp/templates/newrelic.ini.j2]: s/three/five/

you may also want to include this inside the {% if %}. I would expect this to template out such that this comment gets included in the template whether or not the environment is stage. (I think there's also syntax for template comments but I don't remember it offhand)

**Review by matthugs — APPROVED** [review]
This can be tested by building a sandbox against your branch I believe. Time-consuming, but if you're already multitasking not too bad

**Comment by rgraber** [comment]
> This can be tested by building a sandbox against your branch I believe. Time-consuming, but if you're already multitasking not too bad

Will that allow me to see what the file will look like on stage?

**Comment by matthugs** [comment]
> > This can be tested by building a sandbox against your branch I believe. Time-consuming, but if you're already multitasking not too bad
> 
> Will that allow me to see what the file will look like on stage?

not without also changing https://github.com/edx/sandbox-internal/blob/fb16321d0772adee54327d9bccebf0a732ddacc5/ansible/vars/developer-sandbox.yml#L95, good catch. (That, naturally, I would leave to a branch & not try to merge)

**Pull request merged**